yb-voyager/cmd/common.go

```diff
@@ -1269,6 +1270,148 @@ func (ar *AssessmentReport) GetClusterSizingRecommendation() string {
 		ar.Sizing.SizingRecommendation.MemoryPerInstance, ar.Sizing.SizingRecommendation.EstimatedTimeInMinForImport)
 }
 
+func (ar *AssessmentReport) GetTotalTableRowCount() int64 {
+	if ar.TableIndexStats == nil {
+		return -1
+	}
+
+	var totalTableRowCount int64
+	for _, stat := range *ar.TableIndexStats {
+		totalTableRowCount += utils.SafeDereferenceInt64(stat.RowCount)
+	}
+	return totalTableRowCount
+}
+
+func (ar *AssessmentReport) GetTotalTableSize() int64 {
+	if ar.TableIndexStats == nil {
+		return -1
+	}
+
+	var totalTableSize int64
+	for _, stat := range *ar.TableIndexStats {
+		totalTableSize += utils.SafeDereferenceInt64(stat.SizeInBytes)
```

Comment:
similary here, we should add size of of table only

Reply:
added functions: getTableStats() getIndexStats()